jobserver/forms.py

```diff
@@ -135,7 +135,7 @@ def clean(self):
         repo_url = cleaned_data.get("repo")
         branch = cleaned_data.get("branch")
 
-        if not (repo_url and branch):
+        if not (repo_url and branch):  # pragma: no cover
```

Comment:
I'd prefer to see this tested rather than excluded from coverage.

Reply:
Thank you for your comment. I would too. I started writing a test, but I soon realised that I needed to learn more about forms in general as well as forms in job-server, and this issue is (to some degree) time-sensitive. I've made a note to read up, and to return to this line.

Reply:
See #4593. I was hoping to JDI, but haven't managed to. I created an issue before it dropped off my TODO list.